app/components/verkiezingen/generate-rows.js

```diff
@@ -57,7 +57,7 @@ export default class GenerateRowsFormComponent extends Component {
       isBestuurlijkeAliasVan: null,
       beleidsdomein: [],
       status: this.effectiefStatus,
-      publicationStatus: this.publicationStatus,
+      publicationStatus: this.draftPublicationStatus,
```

Comment:
Not sure how but this is changed here https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/344/files

Reply:
Rebased it with master and the change dissapeared ( thats not magic :) )